### DISCUSSION TITLE
Capture and monitor audio

### DISCUSSION THREAD

**Discussion by tariteur** [rocksdanister/lively#2035] [General]
Hello,
I was seeking information on how to capture and monitor audio output generated by various applicationsand if iany code available to assist me in achieving this, can you give it to me pls.

Thanks a lot

**Comment by rocksdanister** [comment]
There is no per application data:
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data

**Reply by tariteur** [reply]
I was curious to understand the code behind it.           "Gets the currently playing audio track from Windows - works with Browser (Chrome), Spotify, Media players.."
